### PR TITLE
checkbox-component-cy-566

### PR DESCRIPTION
**Pull request by steiningermona**

### CODE REVIEW

**Comment by steiningermona** [comment]
**from storybook**
image
Bildschirmfoto 2021-12-14 um 19 16 29
**from figma**
Bildschirmfoto 2021-12-14 um 19 16 59

File: src/components/checkbox/Checkbox.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+import { IconCheckmark } from '../..';
+import { SCheckbox } from './styles/SCheckbox';
+import { SCustomCheckbox } from './styles/SCustomCheckbox';
+
+interface ICheckboxProps {
+  loading?: boolean,
+  checked?: boolean,
+  onChange: (e: Event) => void,
```
Comment (diegomart2000) [inline src/components/checkbox/Checkbox.tsx:9]: 🥳

File: src/icons/IconCheckmark.tsx [hunk 2]
```diff
@@ -2,14 +2,20 @@ import React from "react";
 
 export function IconCheckmark() {
```
Comment (diegomart2000) [inline src/icons/IconCheckmark.tsx]: @steiningermona Please add the size to the interface.

File: src/components/checkbox/Checkbox.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+export const Checkbox: React.FC<ICheckboxProps> = ({ loading, checked, onChange }) => {
+
+  return (
+    <SCheckbox >
+      <SCustomCheckbox onChange={onChange} checked={checked} loading={loading}>
+        <input
+          type="checkbox"
+        />
+        <IconCheckmark></IconCheckmark>
```
Comment (diegomart2000) [inline src/components/checkbox/Checkbox.tsx]: @steiningermona please add the size to conform with the interface.

**Review by diegomart2000 — CHANGES_REQUESTED** [review]

File: src/components/checkbox/Checkbox.tsx (hunk 1 above)
Comment (steiningermona) [reply src/components/checkbox/Checkbox.tsx:9]: 😍

**Review by diegomart2000 — APPROVED** [review]

**Pull request merged**